scaleway/resource_k8s_cluster_test.go

```diff
@@ -241,6 +241,7 @@ func TestAccScalewayK8SCluster_Autoscaling(t *testing.T) {
 }
 
 func TestAccScalewayK8SCluster_OIDC(t *testing.T) {
+	t.Skip("Skip OIDC test, we need to set-up an OIDC server to test it")
```

Comment:
We can reuse one that is existing outside:

- https://developers.google.com/identity/protocols/oauth2/openid-connect

Reply:
We need an account to use an existing one

Reply:
We have plenty :) We can use opensource@scaleway.com for instance.